homeassistant/components/sharkiq/vacuum.py

```diff
@@ -139,9 +139,9 @@ def supported_features(self) -> int:
         return SUPPORT_SHARKIQ
 
     @property
-    def is_docked(self) -> bool | None:
-        """Is vacuum docked."""
-        return self.sharkiq.get_property_value(Properties.DOCKED_STATUS)
+    def is_charging(self) -> bool | None:
```

Comment:
This method is only used once, and contains just a single call.

Maybe move that call directly into the state property method instead.

Reply:
Seems like a good idea, done.